connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorHeartbeatConnector.java

```diff
@@ -48,8 +48,8 @@ public MirrorHeartbeatConnector() {
     @Override
     public void start(Map<String, String> props) {
         config = new MirrorHeartbeatConfig(props);
-        targetAdminClient = config.forwardingAdmin(config.targetAdminConfig());
-        scheduler = new Scheduler(MirrorHeartbeatConnector.class, config.adminTimeout());
+        targetAdminClient = config.forwardingAdmin(config.targetAdminConfig("target-admin"));
```

Comment:
Isn't this a bit vague? Don't we want something like "heartbeats-target-admin" instead?

Reply:
this will be prefixed with the Connector name, with that in mind, it seemed redundant to me, but I can be more specific